src/runtime/profiler_common.cpp

```diff
@@ -378,17 +387,21 @@ WEAK void halide_profiler_report_unlocked(void *user_context, halide_profiler_st
                 }
 
                 sstr << "  " << fs->name << ": ";
-                cursor += 25;
+                cursor += max_func_name_length + 5;
                 while (sstr.size() < cursor) {
                     sstr << " ";
                 }
 
                 float ft = fs->time / (p->runs * 1000000.0f);
+                if (ft < 10000) sstr << " ";
```

Comment:
All "if" statements in Halide should be in {braces}, even trivial single-line ones like these

Reply:
Oh, hmm. Then why doesn't the clang-format CI-step complain?

Reply:
clang-format didn't, but clang-tidy did

Reply:
Ah, what, clang-tidy checks this separately. Clang-format has an option for this too, I believe.